src/helpers/helpers.dom.js

```diff
@@ -150,21 +150,41 @@ export function getMaximumSize(canvas, bbWidth, bbHeight, aspectRatio) {
   };
 }
 
+/**
+ * @param {import('../core/core.controller').default} chart
+ * @param {number} [forceRatio]
+ * @param {boolean} [forceStyle]
+ * @returns {boolean} True if redraw is needed.
+ */
 export function retinaScale(chart, forceRatio, forceStyle) {
-  const pixelRatio = chart.currentDevicePixelRatio = forceRatio || 1;
-  const {canvas, width, height} = chart;
+  const pixelRatio = forceRatio || 1;
+  const devHeight = Math.floor(chart.height * pixelRatio);
+  const devWidth = Math.floor(chart.width * pixelRatio);
+
+  // update css dimensions based on floored device dimentions
```

Comment:
Wrong comment (it's not the css dimensions) and IMO, the comment is useless.

Reply:
Comment removed.